packages/frontend/core/src/modules/collection-rules/impls/group-by/system.ts

```
@@ -0,0 +1,14 @@
+import { Service } from '@toeverything/infra';
+import type { Observable } from 'rxjs';
+
+import type { GroupByProvider } from '../../provider';
+import type { GroupByParams } from '../../types';
+
+export class SystemGroupByProvider extends Service implements GroupByProvider {
+  groupBy$(
+    _items$: Observable<Set<string>>,
```

ESLint

[error] Finnish notation should not be used here.

(rxjs/finnish)